backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -189,7 +189,7 @@ def make_profile_default(self, request: HttpRequest, pk: Any = None) -> Response
             data={"default_profile": profile_manager.profile_id},
         )
 
-    @action(detail=True, methods=["get"])
+    @action(detail=True, methods=["post"])
```

Comment:
@johnyrahul why are we changing this to POST? I think the UI is calling get and this might fail
<img width="1904" alt="image" src="https://github.com/Zipstack/unstract/assets/100672805/94b5a0a9-3ddb-4ddb-8deb-ecdafe1aa50c">

Reply:
The API in screenshot is different!! @ritwik-g . Please check the call immediately above.